### PR TITLE
[music] After adding music directory, ask the user to scan it

### PR DESCRIPTION
**Pull request by razzeee**
Adds a Yes/No dialog after adding a new folder to music. If you choose yes you will trigger a scan of that new path.
@MartijnKaijser 

kodi_change

### CODE REVIEW

File: xbmc/music/windows/GUIWindowMusicBase.cpp [hunk 1]
```diff
@@ -1460,3 +1460,11 @@ void CGUIWindowMusicBase::OnPrepareFileItems(CFileItemList &items)
   if (!items.IsMusicDb())
     RetrieveMusicInfo();
 }
+
+void CGUIWindowMusicBase::OnNewMusicDirectoryAdded(const std::string &path)
+{
+  if (CGUIDialogYesNo::ShowAndGetInput(CVariant{ 13349 }, CVariant{ 20443 }))
+  {
```
Comment (Montellese) [inline xbmc/music/windows/GUIWindowMusicBase.cpp]: No curly braces here (same for the change above).

File: xbmc/dialogs/GUIDialogMediaSource.cpp [hunk 2]
```diff
@@ -395,6 +396,12 @@ void CGUIDialogMediaSource::OnOK()
     {
       CGUIWindowVideoBase::OnAssignContent(share.strPath);
     }
+    else if (m_type == "music" &&
```
Comment (Montellese) [inline xbmc/dialogs/GUIDialogMediaSource.cpp]: I would re-order this to have a `if` condition that checks for `rss://` and `upnp://` and then inside that block the two different conditions for video and music to save some code duplication.

**Comment by MartijnKaijser** [comment]
thx, i think we need to use a different/new string. "Do you want to add this to your library?" or something

**Comment by Razzeee** [comment]
Couldn't find anything better, I'm up for suggestions, that's basically why I posted the screenshot.

**Comment by zag2me** [comment]
Or maybe this: "Do you want to scan this source to your library now?"

**Comment by Razzeee** [comment]
Code updated, strings still the same.

**Pull request closed without merging**